src/v/coproc/wasm_event.h

```diff
@@ -25,11 +25,20 @@
 namespace coproc::wasm {
 
 /// Enumerations for all possible values for keys within the record headers
-enum class event_header { action, description, checksum };
+enum class event_header { action, name, description, checksum };
```

Comment:
`values for keys within the record headers` does this change then break backwards compatibility with existing data on disk?

Reply:
No because that data should already be there, but now that you mention it, this change https://github.com/vectorizedio/redpanda/pull/1607/commits/95603fb3f000f92f524be143e87ccae3eb6ca9e1 does, and it is already in dev

Reply:
im not sure I follow what you are saying. if description had value 1 before, it now has value 2.

Reply:
Yes what I am saying is that the code in this PR only reads the value. The link I pasted in the above comment is the code that adds this second value.